### PR TITLE
New build process

### PR DESCRIPTION
**Pull request by JimBobSquarePants**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp.Web/pulls) open
- [x] I have verified that I am following matches the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description
- Adopts Github Actions as a build process
- Updates dependencies
- Move targets to sensible values.
- Optimize no-command pipline

### CODE REVIEW

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+            - name: Update Codecov
+              uses: iansu/codecov-action-node@v1.0.0
+              if: matrix.options.codecov == true
+              with:
+                  token: ${{secrets.CODECOV_TOKEN}}
+                  file: "coverage.${{matrix.options.framework}}.xml"
+                  flags: unittests
+
+            - name: Pack # We can use this filter as we know it happens only once and takes the most time to complete.
```
Comment (tocsoft) [inline .github/workflows/build-and-test.yml]: we should be using the split build/publish workflow here not skip and hope it takes the longest

File: GitVersion.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+continuous-delivery-fallback-tag: ci
+branches:
+    master:
+        tag: unstable
```
Comment (tocsoft) [inline GitVersion.yml]: isn't this missing the `ContinuousDeployment` mode?

File: .github/workflows/build-and-test.yml (hunk 1 above)
Comment (JimBobSquarePants) [reply .github/workflows/build-and-test.yml]: Yeah, good spot. I'll update that once I get it working.

File: GitVersion.yml (hunk 2 above)
Comment (JimBobSquarePants) [reply GitVersion.yml]: Cripes! Fixed

**Pull request merged**